tests/test_client.py

```diff
@@ -1,307 +1,31 @@
-    jobs = await client.jobs()
-    for job in jobs:
-        if job.Port != 80:
-            await client.kill_job(job.ID)
-    assert len(await client.jobs()) <= 2
-
-
-@test("Client can generate an MSF stager", tags=["client"])
-async def _(client: SliverClient = sliver_client):  # type: ignore
-    stager = await client.generate_msf_stager(
-        arch="amd64",
-        format="raw",
-        host="127.0.0.1",
-        port=9000,
-        os="windows",
-        protocol=StageProtocol.TCP,
-        badchars=[],
-    )
-    assert Path(stager.File.Name)
-
-
-@test("Client can generate Donut shellcode", tags=["client"])
-async def _(client: SliverClient = sliver_client, data_dir: Path = data_dir):  # type: ignore
-    dll_data = Path(data_dir / "test_write.exe").read_bytes()
-    assert await client.shellcode(dll_data, "Main")
-
-
-@test("Client can interact with a session", tags=["client"])
-async def _(client: SliverClient = sliver_client):  # type: ignore
-    sessions = await client.sessions()
-    session = sessions[0]
-    assert await client.interact_session(session.ID)
-
-
-@test("Client can interact with a beacon", tags=["client"])
-async def _(client: SliverClient = sliver_client):  # type: ignore
-    beacons = await client.beacons()
-    beacon = beacons[0]
-    assert await client.interact_beacon(beacon.ID)
-
-
-@test("Client can add website content", tags=["client"])
-async def _(client: SliverClient = sliver_client, data_dir: Path = data_dir):  # type: ignore
-    html_content = Path(data_dir / "website.html").read_bytes()
-    assert await client.add_website_content(
-        "sliverpy-test", "sliverpy", "test/html", html_content
-    )
-
-
-@test("Client can update website content", tags=["client"])
-async def _(client: SliverClient = sliver_client, data_dir: Path = data_dir):  # type: ignore
-    html_content = Path(data_dir / "website_update.html").read_bytes()
-    assert await client.add_website_content(
-        "sliverpy-test", "sliverpy", "test/html", html_content
-    )
-
-
-@test("Client can list websites", tags=["client"])
-async def _(client: SliverClient = sliver_client):  # type: ignore
-    assert "sliverpy-test" in [website.Name for website in await client.websites()]
-
-
-@test("Client can remove website content", tags=["client"])
-async def _(client: SliverClient = sliver_client):  # type: ignore
-    assert await client.remove_website_content("sliverpy-test", ["sliverpy"])
-
-
-@test("Client can remove website", tags=["client"])
-async def _(client: SliverClient = sliver_client):  # type: ignore
-    await client.remove_website("sliverpy-test")
-    assert "sliverpy-test" not in [website.Name for website in await client.websites()]
+@test("Client can get job by port", tags=["client", "server_info"])
+async def _(client: SliverClient = sliver_client, test_const: TestConstants = test_constants):  # type: ignore
+    assert await client.job_by_port(test_const.multiplayer_job_port)

```

Comment:
The multiplayer port isn't considered a "job" (although it's a job on the server), so this test fails. We can use `jobs[0].Port`  though